release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1353,6 +1353,11 @@ As part of the continued deprecation of the Operator package manifest format, th
 
 *Machine Config Operator*
 
+* When upgrading from 4.6 to 4.7, the host name set by the *vsphere-hostname* service was applied only when a node is installed. If the host name was not statically set prior to upgrading, the host name could be lost. This update removes the condition which allowed the *vsphere-hostname* service to run only when a node is installed. As a result, vSphere host names are no longer lost when upgrading. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1942207[*BZ#1942207*])
```

Comment:
```suggestion
* When upgrading from 4.6 to 4.7, the host name set by the *vsphere-hostname* service was applied only when a node was installed. If the host name was not statically set prior to upgrading, the host name could be lost. This update removes the condition which allowed the *vsphere-hostname* service to run only when a node is installed. As a result, vSphere host names are no longer lost when upgrading. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1942207[*BZ#1942207*])
```
I thought a service name would be in backticks rather than bold, but I'm not sure.